EFRS-808: Review Readme.md file (#181)
exadel-inc/CompreFace: README.md

Creates an example of the face by saving images. To train the system, you can add as many images as you want.

```http request

> curl  -X POST "http://localhost:8000/api/v1/faces/?subject=<face_name>" \

Reviewer: Let's rename all `face_name` to `subject` and `face_id` to `image_id`

-H "Content-Type: multipart/form-data" \

> -H "x-api-key: <model_api_key>" \

Reviewer: Everywhere
```suggestion
-H "x-api-key: <faces_collection_api_key>" \
```

### Recognize faces from given image

Recognizes faces from the uploaded images.
```http request

> curl  -X POST "http://localhost:8000/api/v1/recognize?limit=<limit>&prediction_count=<prediction_count>" \

Author: Nope. Now it's a constant value `0.5D`

> curl  -X POST "http://localhost:8000/api/v1/recognize?limit=<limit>&prediction_count=<prediction_count>" \

Reviewer: But why?) how we chose the value?

> curl  -X POST "http://localhost:8000/api/v1/recognize?limit=<limit>&prediction_count=<prediction_count>" \

Author: It's not my decision, but Maryna's. I think, we should add threshold in an another task.

### Recognize faces from given image

> Recognizes faces from the uploaded images.

Reviewer: ```suggestion
Recognizes faces from the uploaded image.
```

### Verify faces from given image

> Compares faces from the uploaded images with provided face id.

Reviewer: ```suggestion
Compares faces from the uploaded image with face in saved image id.
```

```http request

> curl  -X POST "http://localhost:8000/api/v1/faces/<image_id>/verify?limit=<limit>" \

Author: Nope. Now it's a constant value `0.5D`
